h5pt.go

```diff
@@ -23,6 +23,14 @@ type Table struct {
 	Identifier
 }
 
+type appendData struct {
+	ptr       unsafe.Pointer
+	memSize   C.size_t
+	offset    C.size_t
+	noRecords C.size_t
```

Comment:
I would rename this `numRecords` or `nrecords`.
`noRecords` reads as it's "no (such) record".